mate-panel/panel-menu-items.c

```diff
@@ -568,8 +568,8 @@ drive_poll_for_media_cb (GObject      *source_object,
 		g_error_free (error);
 	}
 
-	//FIXME: should we mount the volume and activate the root of the new
-	//mount?
+	/*FIXME: should we mount the volume and activate the root of the new
+	 * mount? */
```

Comment:
Can you please move the closing element `*/` to the next line?
Same style as all others.